### ISSUE TITLE
New features

### ISSUE THREAD

**Issue by xinyuejohn** [theislab/ehrdata#3]
Description of feature

This data structure is supposed to easily support the task for EHR analysis:

# Data Subsetting
## Cohort selection
e.g. Patient with certain disease
## Certain feature selection
e.g. Only use the first 48 hours of the ICU record

# Data Processing
## Tabular data
Data validation, Encode, Sampling, Discretization, Standardization, Outlier detection

# Data pairing between different modalities 
e.g. between time series tabular data and image date

# Data Split
e.g. For training set and test set split

Some helpful functions
# Need to classify some preprocessing functions in EHR data pipeline and extract some common one
